tests/airgapped/airgap.sh

```diff
@@ -79,6 +79,26 @@ function setup_microk8s() {
   airgap_wait_for_pods "$NAME"
 }
 
+function create_ingress_object_in_kubeflow_namespace() {
+  local NAME=$1
+  lxc exec "$NAME" -- bash -c 'echo "apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: http-ingress
+  namespace: kubeflow
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: istio-ingressgateway-workload
+            port:
+              number: 80" | microk8s kubectl apply -n kubeflow -f -'
+}
+
```

Comment:
```suggestion
function create_ingress_object_in_kubeflow_namespace() {
  local NAME=$1
  if ! lxc exec "$NAME" -- bash -c 'cat <<EOF | microk8s kubectl apply -n kubeflow -f -
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: http-ingress
  namespace: kubeflow
spec:
  rules:
  - http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: istio-ingressgateway-workload
            port:
              number: 80
EOF'; then
    echo "Error: Failed to create Ingress object in Kubeflow namespace"
    return 1
  fi
}
```

Maybe we can add some error catching with message if the command fails. Something like this.

Reply:
+1